examples/features/metadata_interceptor/server/main.go

```diff
@@ -49,10 +49,22 @@ func unaryInterceptor(ctx context.Context, req any, info *grpc.UnaryServerInfo,
 		return nil, errMissingMetadata
 	}
 
+	// create and set metadata from interceptor to server
```

Comment:
Comments that are complete sentences should be capitalized and punctuated like standard English sentences. Here and elsewhere in this PR.

See: https://google.github.io/styleguide/go/decisions#comment-sentences

Reply:
Done